integration/container/run_linux_test.go

```diff
@@ -57,6 +58,11 @@ func TestNISDomainname(t *testing.T) {
 	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.40"), "skip test from new feature")
 	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
 
+	// Rootless supports custom Hostname but doesn't support custom Domainname
+	//  OCI runtime create failed: container_linux.go:349: starting container process caused "process_linux.go:449: container init caused \
+	//  "write sysctl key kernel.domainname: open /proc/sys/kernel/domainname: permission denied\"": unknown.
+	skip.If(t, testEnv.IsRootless, "to be investigated")
```

Comment:
Could you make this a `TODO`, and add a tracking issue for this (and add issue/link in the comment?)

Reply:
issue; https://github.com/moby/moby/issues/40632

Reply:
added